### ISSUE TITLE
[Feature] getCldVideoUrl examples

### ISSUE THREAD

**Issue by colbyfayock** [cloudinary-community/astro-cloudinary#29]
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

Add examples to the getCldVideoUrl documentation page.

## **Describe the solution you'd like**

Replicate all of the examples on CldVideoPlayer, but utilizing the URL construction function, passing it off to the native `<video` element.

See  getCldImageUrl for similar examples.

**Comment by RaghavMangla** [comment]
Hi, I'd like to work on this issue

**Comment by saai-syvendra** [comment]
Hi @colbyfayock! Shall I work on this?

**Comment by colbyfayock** [comment]
sure!

**Comment by saai-syvendra** [comment]
@colbyfayock I believe only the basic usage and cropping&resizing example can be replicated from the CldVideoPlayer... As the other examples are specific to the video player... Let me know if I'm mistaken...

And where can I find the possible options that can be passed to getCldVideoUrl

**Comment by colbyfayock** [comment]
You're right about the examples, perhaps you could add an additional example for a text overlay? should work similar to this: https://astro.cloudinary.dev/guides/text-overlays

getCldVideoUrl uses essentially the same API as the image version. it needs documentation though: https://astro.cloudinary.dev/cldimage/configuration
